.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": ["@babel/plugin-proposal-class-properties", "@babel/plugin-transform-runtime"]
```

Comment:
Are you intending to use class syntax? `@babel/plugin-proposal-class-properties`